static/js/pages/instances/InstanceList.tsx

```diff
@@ -237,20 +258,17 @@ const InstanceList: FC = () => {
               items={[
                 <StartStopInstanceBtn
                   key="startstop"
-                  className="u-instance-actions"
+                  className={classnames(
+                    "u-instance-actions",
+                    starting.concat(stopping).includes(instance.name) &&
+                      "u-hide"
```

Comment:
I suggest: `"u-hide": [...starting, ...stopping].includes(instance.name)`
Shorter and easier to read and understand.

Reply:
Yeah, that is better indeed.

Reply:
I kept the `concat` version, because prettier allows it on fitting on one line. But the `:` version I adopted, because that is easier to read.